File: 프로그래머스/1/159994. 카드 뭉치/카드 뭉치.py
def solution(cards1, cards2, goal):
    for item in goal:
        if len(cards1)!=0 and cards1[0]==item:
            cards1.remove(item)
        elif len(cards2)!=0 and cards2[0]==item:
            cards2.remove(item)
        else:
            return "No"
    return "Yes"

File: 프로그래머스/1/159994. 카드 뭉치/카드 뭉치.js
function solution(cards1, cards2, goal) {
    let result="Yes";
    goal.forEach(item=>{
        if(cards1.length!=0 && item==cards1[0]){
            cards1.splice(0,1);
            console.log(cards1);
        }
        else if(cards2.length!=0 && item==cards2[0])
            cards2.splice(0,1);
        else
            result="No";
    })
    return result;
}